### DISCUSSION TITLE
Deepstack detection Night time

### DISCUSSION THREAD

**Discussion by RejEdm** [djdd87/SynoAI#116] [Ideas]
Hello djdd87, I have been using SynoAI for over a month and it's been super reliable and I wanted to say thank you :)

I have noticed at night time, deepstack is not as effective at detecting... I do realize you can't do anything about deepstack, but I have read about deepstack exdark. It improve night time detection. Some folks using Blue Iris have both deekstack and deepstack EXdark running one profile for daytime and the other for night time.

I am wondering if it would be possible to have deepstack and deepstack exdark running and configured in the app settings.json? Having deepstack run during the day and deepstack exdark run at night?

I do apologize if this has been requested/discussed in the past, I have looked around and didn't find anything on this topic.

**Comment by djdd87** [comment]
Raise a feature request via an issue. I suggest we have the ability to define the additional deepstack exdark container and the active hours in which they should run. There'd be an issue there though potentially with fixed hours, because the times of daylight obvious change continually.
